python/setup.py.in

```diff
@@ -181,6 +181,14 @@ else:
     command = "patchelf --set-rpath '$ORIGIN/../libs/' ${PADDLE_BINARY_DIR}/python/paddle/fluid/core.so"
 if os.system(command) != 0:
     raise Exception("patch core.so failed, command: %s" % command)
+if '${WITH_FLUID_ONLY}'== 'OFF':
```

Comment:
> Use @ WITH_FLUID_ONLY@? not quiet sure, it's a cmake template, can cmake update ${} also?

Yes, `${WITH_FLUID_ONLY}` is already used in `setup.py.in`, such as 
https://github.com/PaddlePaddle/Paddle/blob/74fa603cce9645140ea21bdbef3be9851a623f50/python/setup.py.in#L127
https://github.com/PaddlePaddle/Paddle/blob/74fa603cce9645140ea21bdbef3be9851a623f50/python/setup.py.in#L135

> And maybe add this as a function, since it's called several times.

Can I refine it in next PR?

Reply:
I see. Approving.